wiki/pages/How_to_visualize.md

```diff
@@ -0,0 +1,17 @@
+#What does it mean to visualize a simulation?
+
+An advantage of simulating materials is that we can learn, not only about 
+how a material will settle given, for example, the number of particles, density, and temperature,
+but also how the system evolved at every step from beginning to end. With every atom's position tracked
+throughout the course of the simulation, the first investigation into the nature of the system 
+is often to the play the "movie" of where the atoms went and who they fraternized with along the way.
+
+
+## Tools for the job.
+
+The cmelab is currently utilizing two distinct resources for this purpose. They both have their pros/cons. 
+
+1. [VMD](http://www.ks.uiuc.edu/Research/vmd/) is an open source software package. The interface may be unintuitive to begin, but our lab has more expertise with this tool at present. 
```

Comment:
nice, let's mention the vmdrc file included in the config_files directory of this repo

Reply:
we might also want to mention that more detailed install instructions for vmd and the gsd plugin are in `pages/Setting_up_an_OSX_Computer.md` and `pages/Installing_the_Frequently-Used_Programs_for_CME-Lab.md`